assets/javascripts/modules/checkout/emailCheck.js

```diff
@@ -0,0 +1,12 @@
+define(['utils/ajax'], function (ajax) {
+    'use strict';
+
+    return function(email) {
+        return ajax({ url: '/checkout/check-identity?email=' + email }).then(function (response) {
+            return response.emailInUse;
+        }).fail(function (err) {
```

Comment:
See below, should be handled in `validations.js`, just forgot to re-add that part…